vadim.martsun/task_4/src/main/java/config/AppConfig.java

```diff
@@ -0,0 +1,35 @@
+package config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import random.RandomBankAccountService;
+import random.RandomBankAccountServiceImpl;
+import service.BankService;
+import service.BankServiceImpl;
+import service.TestBankService;
+import service.transactionExplorer.TransactionExplorer;
+import service.transactionExplorer.TransactionExplorerImpl;
+
+@Configuration
+public class AppConfig {
+
+    @Bean(name = "bankService")
```

Comment:
You already using @Service annotation to create component. You don't need to duplicate @Bean inside @Configuration